appveyor.yml

```diff
@@ -0,0 +1,41 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\puli\repository
+
+environment:
+  matrix:
+    - COMPOSER_FLAGS: ""
+    - COMPOSER_FLAGS: --prefer-lowest --prefer-stable
+
+install:
+  - cinst -y OpenSSL.Light
+  - SET PATH=C:\Program Files\OpenSSL;%PATH%
+  - cinst -y php
+  - cd c:\tools\php
+  - copy php.ini-production php.ini
+  - echo date.timezone="UTC" >> php.ini
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_bz2.dll >> php.ini
+  - echo extension=php_curl.dll >> php.ini
+  - echo extension=php_fileinfo.dll >> php.ini
+  - echo extension=php_exif.dll >> php.ini
+  - echo extension=php_gd2.dll >> php.ini
+  - echo extension=php_intl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_mysqli.dll >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_pdo_mysql.dll >> php.ini
+  - echo extension=php_soap.dll >> php.ini
+  - echo extension=php_sockets.dll >> php.ini
+  - echo extension=php_xmlrpc.dll >> php.ini
+  - echo extension=php_xsl.dll >> php.ini
+  - echo memory_limit=512M >> php.ini
+  - SET PATH=c:\tools\php;%PATH%
+  - cd c:\projects\puli\repository
+  - php -r "readfile('http://getcomposer.org/installer');" | php
+  - php composer.phar update %COMPOSER_FLAGS% --no-interaction
```

Comment:
you should add ``--no-progress`` because the appveyor logs don't support overwritting the output and so the composer output looks bad when it tries to display the progress